### ISSUE TITLE
Store data in LocalStorageData

### ISSUE THREAD

**Issue by djmisterjon** [englercj/playground#17]
It would be great if you can add the **Slug value** or data to the **LocalStorage**.
![image](https://user-images.githubusercontent.com/24865815/73509225-6b308800-43ac-11ea-87db-134d659e71a6.png)

I would like to make a little script to store and listing **LocalStorageData** in an excel vba sheets.
Easy way to view ,sort, preview a list without an account logging system.
I would have liked to add the feature by myself, but it's ts. :)

**Comment by englercj** [comment]
Not 100% sure I follow. Why do you want me to put the slug into local storage?

**Comment by djmisterjon** [comment]
because I find it boring to copy paste my lists, and sometime they disappear ! Slack without premium.
![image](https://user-images.githubusercontent.com/24865815/73599670-e791ab00-4513-11ea-8193-d2342f00ad3b.png)
So i want make a script that get local strogage data , make connection , make screenshot and store in Excel sheets.

**Comment by djmisterjon** [comment]
or maybe if you have another way to make or share public data of the sql db ?

**Comment by englercj** [comment]
So you want the site to store a list in your localstorage of each entry you make, so that you can export them to excel.

Why not paste the links into excel, or a text document? I don't like the idea of putting urls in everyone's local storage just for your script to consume. I'd rather just build the right APIs to support ownership of playgrounds.

**Comment by djmisterjon** [comment]
yes, in fact I thought it was an abandoned idea because too difficult.
I was just trying to offer you a quick and efficient temporary solution, but I don't like the idea me too, but these better than nothing. :)

Because I use your application a lot to share help with noobs on Discords , but these just very difficult to manage all example when need share or search. :)
We really need a way to save with list or share.
I know it a very hard job and your very busy, and it why i ask you simple solution.
But if you haven't given up on the idea of a profile db, ignore my ask. :)

**Comment by englercj** [comment]
> I thought it was an abandoned idea because too difficult.

Nope, just haven't had time to do it yet. I still plan to get it done at some point.

**Comment by LeedaCode** [comment]
Or you can join  account ？, in github login? List the  creation page for each account. @englercj

**Comment by englercj** [comment]
That is the plan, just need time to do it.
